client/src/components/Home.jsx

```diff
@@ -274,21 +245,29 @@ const Home = ({ setIsLoading, isMobile, isLoggedIn, isLoading, setIsLoggedIn, sh
     },
     [draggedEvent, setDraggedEvent, newEvent]
   )
+
   // All Components
+
+  const naviBar = (<TopBar isLoading={isLoading} setIsLoggedIn={setIsLoggedIn} isLoggedIn={isLoggedIn} isMobile={isMobile} onCalendar={onCalendar} setOnCalendar={setOnCalendar} userEmail={userEmail}/>);
+
+  const toDoList = (<ToDoList isMobile={isMobile} taskData={myEvents.flat()} draggedEvent={draggedEvent} setDraggedEvent={setDraggedEvent} handleDragStart={handleDragStart} addCategory={addCategory} updateTodo={updateTodo} addTodo={addTodo} deleteTodo={deleteTodo}/>);
+
+  const myCalendar = (<MyCalendar myEvents={myEvents} moveEvent={moveEvent} resizeEvent={resizeEvent} changeTitle={changeTitle} onDropFromOutside={onDropFromOutside}/>);
+
```

Comment:
Not sure I agree with this.  As with objects, is better to just continue on the next line instead of letting it scroll off the side of the screen.  Putting a new line in between long variable declarations doesn't really help in this situation of styling.

Reply:
It was a long one, so I didn't really want to touch it, but needed some separation for readability at least.

Reply:
Fair enough.  Yeah, I'm not one for messing with code I didn't write if it's not broken!